File: Euler-Particles/src/color/RGB/CyanColorGenerator.java
package color.RGB;

import processing.core.PApplet;

public class CyanColorGenerator extends RGBColorGenerator {

    public CyanColorGenerator(PApplet p) {
        super(p);
    }

    public int randomColor() {
        p.colorMode(p.RGB, 255);
        int r = (int)p.random(50, 100);
        int g = (int)p.random(100, 255);
        int b = (int)p.random(100, 255);
        return p.color(r, g, b);
    }

}

File: docs/Acceleration/color/rgb/cyan_color_generator.js
class CyanColorGenerator extends RGBColorGenerator {

    getRandomColor() {
        colorMode(RGB, 255);
        let r = int(random(50, 100));
        let g = int(random(100, 255));
        let b = int(random(100, 255));
        return color(r, g, b);
    }
}
